### PR TITLE
alert feature implemented

### PR DESCRIPTION
**Pull request by bharatjain7723**
### Change Log

Added Alert Component

- Primary
- Secondary
- Success
- Info
- Warning
- Danger

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+:root {
+  --alert-primary: 13, 110, 253; /* #0d6efd3 */
+  --alert-secondary: 108, 117, 125; /* #6c757d */
+  --alert-success: 25, 135, 84; /* #198754 */
+  --alert-info: 13, 202, 240; /* #0dcaf0 */
+  --alert-warning: 255, 193, 7; /* #ffc107 */
+  --alert-danger: 220, 53, 69; /* #dc3545 */
+  --alert-background-alpha: 0.5;
+  --alert-border-alpha: 0.25;
+}
```
Comment (Creator-singh) [inline Components/Alert/alert.css]: Make separate file for utilities.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  font-family: "Roboto", sans-serif;
+}
+
+.component-container {
+  margin: 40px 20px;
+}
+
+.component-heading {
+  margin-bottom: 20px;
+}
```
Comment (Creator-singh) [inline index.css]: use rem where ever possible. if you are using font-size: 62.5% then use px only when size is less than 10px else use rem where ever possible.

**Review by Creator-singh — CHANGES_REQUESTED** [review]
make the changes else everything looks good.

**Review by hiteshverma27 — APPROVED** [review]
Try to use rems else it looks good.

File: index.css (hunk 2 above)
Comment (bharatjain7723) [reply index.css]: done

File: Components/Alert/alert.css (hunk 1 above)
Comment (bharatjain7723) [reply Components/Alert/alert.css]: done

File: index.css (hunk 2 above)
Comment (bharatjain7723) [reply index.css]: @Creator-singh pls review again

**Review by Creator-singh — APPROVED** [review]
great work

**Pull request merged**